TicketMate.Api/Controllers/ProjectController.cs

```diff
@@ -2,6 +2,8 @@
 using TicketMate.Application.Abstraction;
 using TicketMate.Application.Requests.ProjectRequests.Insert;
 using TicketMate.Application.Requests.ProjectRequests.GetByGuid;
+using TicketMate.Application.Requests.ProjectRequests.DeleteByGuid;
+using TicketMate.Persistence.DataRequestObjects.ProjectRequests;

```

Comment:
I don't think the API layer (ProjectController) needs to reference the Persistence.DataRequestObjects.ProjectRequests - can we check if this can be removed?